Copies Atlas CLI Kubernetes Plugin commands for release tag: v1.1.0 (#949)
mongodb/docs-atlas-cli: source/command/atlas-kubernetes-dry-run.txt

.. _atlas-kubernetes-dry-run:

========================
atlas kubernetes dry-run
========================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> Deploy and run Atlas Kubernetes Operator in dry-run mode

Reviewer: ```suggestion
Deploy and run Atlas Kubernetes Operator in Dry Run mode.
```

> This command deploys the Atlas Kubernetes operator with the DryRun mode.

Reviewer: ```suggestion
This command deploys the Atlas Kubernetes Operator with the Dry Run mode.
```

Syntax
------

.. code-block::
   :caption: Command Syntax

   atlas kubernetes dry-run [options]

.. Code end marker, please don't delete this comment

Options
-------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - -h, --help
     - 
     - false

>      - help for dry-run

Reviewer: ```suggestion
     - help for Dry Run
```

   * - --operatorVersion
     - string
     - false
     - Version of Atlas Kubernetes Operator to generate resources for. This value defaults to "2.7.0".
   * - --orgId
     - string
     - false
     - Organization ID to use. This option overrides the settings in the configuration file or environment variable.
   * - --targetNamespace
     - string
     - false

>      - Namespaces to use for generated kubernetes entities

Reviewer: ```suggestion
     - Namespaces to use for generated Kubernetes entities.
```
